index.html

```diff
@@ -45,7 +45,7 @@
         </div>
 
         <div>
-          <img src="/images/screenshot.gif" />
+          <img src="./images/screenshot.gif" />
```

Comment:
I think this depends on how you're testing the site. Can you tell me about how this addresses your issue?